style(linter/plugins): update `oxfmt` config to format `oxlint` fixtures

overlookmotel opened the pull request:
Update `oxfmt` config so it formats `apps/oxlint/test/fixtures`.

Actual formatting changes were in PRs #15600 and #15674. This PR alters the config so all these files will continue to be formatted in future.

leaysgur reviewed (commented):
According to #15600, you want to format

- `apps/oxlint/test/fixtures`

right?

In case of that, this seems to work.

```diff
diff --git a/oxfmtrc.jsonc b/oxfmtrc.jsonc
index 36bdccee3..07e96aa2a 100644
--- a/oxfmtrc.jsonc
+++ b/oxfmtrc.jsonc
@@ -7,6 +7,7 @@
     "**/tests/**",
     "**/generated/**",
     "**/fixtures/**",
+    "!apps/oxlint/test/fixtures/**",
     "tasks/coverage/node-compat-table",
     "tasks/coverage/misc",
     "tasks/coverage/src/runtime/babelHelpers.js",

```

What you were trying in your opening comment appears:

```json
{
  "ignorePatterns": [
    "**/fixtures/**",
    "!apps/oxlint/fixtures/**",
  ]
}
```

- `apps/oxlint/fixtures`

should be

- `apps/oxlint/test/fixtures`

overlookmotel committed:
style(linter/plugins): update `oxfmt` config to format `oxlint` fixtures (#15601)

Update `oxfmt` config so it formats `apps/oxlint/test/fixtures`.

Actual formatting changes were in PRs #15600 and #15674. This PR alters the config so all these files will continue to be formatted in future.
